Resources/Prototypes/_Goobstation/Heretic/GameRules/roundstart.yml

```
@@ -22,4 +20,4 @@
       lateJoinAdditional: true
       mindRoles:
       - MindRoleHeretic
-      startingGear: HereticGear # see Roles/Antags/heretic.yml
\ No newline at end of file
+      startingGear: HereticGear # see Roles/Antags/heretic.yml
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)